src/components/InstanceCard/InstanceCard.tsx

```diff
@@ -56,6 +67,7 @@ export const InstanceCard: React.FC<{
         </div>
         <div>{EditionCardUtils.getWorldCatElem(instance)}</div>
         <Link to="/license">{EditionCardUtils.getLicense(previewItem)}</Link>
+        {isPhysicalEdition && EditionCardUtils.getScanAndDeliverBlurb()}
```

Comment:
I know this is a pattern we've been using throughout DRB, but I recommend refactoring this to explicitly render a component instead of a function that returns JSX. This makes our components more scalable for the future, and it is also a slight performance improvement.

`{isPhysicalEdition && <ScanAndDeliverBlurb />}`

For this PR, let's ensure all new components are using this method. If there are other instances throughout the code where we are returning JSX from a function instead of rendering the component directly, let's make a ticket to update that so that it matches the recommended pattern.